pandas/core/groupby/generic.py

```diff
@@ -303,12 +310,12 @@ def _aggregate_multiple_funcs(self, arg):
 
             # indicated column order
             columns = next(zip(*arg))
+
         else:
             # list of functions / function names
             columns = []
             for f in arg:
                 columns.append(com.get_callable_name(f) or f)
-
```

Comment:
Can you revert adding/removing these blank lines?

Reply:
Yes, will fix this.